template/user.deactivation/body.html

```diff
@@ -50,14 +50,17 @@
   ></div>
 
   <p>
-    You are receiving this email because you have an OpenShift.io Online
-    free account ({{ .custom.userEmail }}) and your action is required to
-    keep it active.
+    You are receiving this email because you have a free online <a href="https://openshift.io">OpenShift.io</a> account
+    associated with {{ .custom.userEmail }}
   </p>
 
   <p>
-    Login into the system by <strong>{{ .custom.expiryDate }}</strong> in order to continue
-    using OpenShift.io uninterrupted.
+    Action is required to keep your account active.
+  </p>
+
+  <p>
+    To keep your account active, please log in by <strong>{{ .custom.expiryDate }}</strong> to ensure
+    uninterrupted use of <a href="https://openshift.io">OpenShift.io</a>
```

Comment:
the content above does not match with what was requested, AFAICS:
```
...
To keep your account active, please log in by Monday, June 17 to ensure uninterrupted use of OpenShift.io 

OpenShift.io | Che.OpenShift.io
...
```

but to be honest, I'm not sure why we have this `OpenShift.io | Che.OpenShift.io` paragraph in the middle of the mail body 🤔

Reply:
Hm.. What did I miss exactly? That `OpenShift.io | Che.OpenShift.io` thing in the email with the desired text too.

Reply:
ah my bad, it was above, in a collapsed section (because there was not change compared to previous version) 😬